custom/icds_reports/static/js/directives/mobile-datepicker/mobile-datepicker.directive.js

```diff
@@ -309,22 +310,60 @@ angular.module('icdsApp').directive('mobileDatepicker', function() {
 
             })(jQuery);
 
+            //this method returns months to be displayed corresponding to the year selected
+            // if its minYear
+            getItems = function () {
```

Comment:
is this overriding another `getItems` function? I'm surprised it doesn't have a `var` declaration.

Reply:
also, if I'm understanding it correctly I think a better name would be `getAvailableMonthsForSelectedYear`

Reply:
> is this overriding another `getItems` function? I'm surprised it doesn't have a `var` declaration.

mistake on my part. I copied function down and forgot to remove the one that was already defined